data/json/recipes/chem/chemicals.json

```diff
@@ -39,6 +39,159 @@
       [ [ "chem_glycerol", 20 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "chem_acetone",
+    "charges": 2,
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "skill_used": "chemistry",
+    "difficulty": 4,
+    "time": "1 h",
+    "batch_time_factors": [ 80, 2 ],
+    "autolearn": true,
```

Comment:
I don't think any of these could be autolearn 🤔

*Does not apply to Innawoods

Reply:
Done.